main.go

```diff
@@ -11,13 +12,16 @@ import (
 	"github.com/ONSdigital/dp-frontend-cookie-controller/config"
 	"github.com/ONSdigital/dp-frontend-cookie-controller/routes"
 	health "github.com/ONSdigital/dp-healthcheck/healthcheck"
+	dphttp "github.com/ONSdigital/dp-net/v2/http"
+	dpotelgo "github.com/ONSdigital/dp-otel-go"
 	render "github.com/ONSdigital/dp-renderer/v2"
 	"github.com/ONSdigital/dp-renderer/v2/middleware/renderror"
 	"github.com/ONSdigital/log.go/v2/log"
 	"github.com/gorilla/mux"
 	"github.com/justinas/alice"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
 
-	dpnethttp "github.com/ONSdigital/dp-net/v2/http"
```

Comment:
This seems to be removing an import that's added a few lines above with a different alias. The net effect is just to rename the alias, which feels like an unnecessary change

Reply:
Renamed back to the original alias